src/platform/plugins/shared/chart_expressions/expression_xy/public/components/xy_chart.tsx

```diff
@@ -787,11 +791,13 @@ export function XYChart({
               actions={getTooltipActions(
                 dataLayers,
                 onClickMultiValue,
+                onCreateAlertRule,
                 fieldFormats,
                 formattedDatatables,
                 xAxisFormatter,
                 formatFactory,
-                interactive && !args.detailedTooltip && !isEsqlMode
+                isEsqlMode,
+                interactive && !args.detailedTooltip
```

Comment:
This is going to activate everywhere the Lens ES|QL chart is emdedded (for example in Discover). Do we want this or do we want this only when embedded in dashboards?

Reply:
We could restrict it to dashboards. Is there a good way of finding out what the chart is embedded in from the XYChart component? Something we need to pass down from the embeddable maybe?

Reply:
This is done!